org.eclipse.lemminx/src/main/java/org/eclipse/lemminx/services/XMLCompletions.java

```diff
@@ -753,6 +754,18 @@ private void collectInsideContent(CompletionRequest request, CompletionResponse
 			collectCloseTagSuggestions(tagNameRange, true, true, false, request, response);
 		}
 		// Participant completion on XML content
+		if (request.getNode() instanceof DOMElement) {

```

Comment:
I think this section about DOMElement probably should be removed, I don't think it actually addresses the issue that was reported (even though it made downstream lemminx-maven workaround tests pass..). 

The original bug was about triggering a completion on text, not on a node?